database/ddl/createTables.sql

```diff
@@ -29,6 +29,8 @@ create table ugt.incident
     created_on         timestamp with time zone not null
 );
 
+CREATE INDEX idx_incident_timestamp ON ugt.incident(incident_timestamp);
```

Comment:
Should we also include the incident_type in the index (we allow to filter by it)?